app/src/main/java/org/mozilla/focus/fragment/HomeFragment.kt

```diff
@@ -75,13 +78,17 @@ class HomeFragment : Fragment() {
         val homeTiles = mutableListOf<HomeTile>()
         val inputAsString = context.assets.open(HOME_TILES_JSON_PATH).bufferedReader().use { it.readText() }
         val jsonArray = JSONObject(inputAsString).getJSONArray(HOME_TILES_JSON_KEY)
+        val userDetails = context.getSharedPreferences(HOME_TILES_PREFS, MODE_PRIVATE)
+        val blacklist = userDetails.getStringSet(BLACKLIST, null)
```

Comment:
nit: I would pull this whole part of making a list of HomeTiles into a function so it doesn't clutter up the init (and make it hard to read), something that takes the sharedprefs and home tiles jsonArray, and returns a list of HomeTiles, with the blacklist filtered out.